### PR TITLE
Feat: Compliance Triggers and Customers

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Review Customer screen + Compliance triggers](https://trello.com/c/IgROorRd/130-review-customer-screen-compliance-triggers)

File: lib/customers.js [hunk 1]
```diff
@@ -454,7 +454,7 @@ function getCustomerById (id) {
     c.id_card_data, c.id_card_data_override, c.id_card_data_expiration, c.id_card_photo_path,
     c.id_card_photo_override, t.tx_class, t.fiat, t.fiat_code, t.created, 
     row_number() over (partition by c.id order by t.created desc) as rn,
-    count(0) over (partition by c.id) as total_txs,
+    sum(case when t.id is not null then 1 else 0 end) over (partition by c.id) as total_txs,
```
Comment (joshmh) [inline lib/customers.js:457]: Can we filter by NOT NULL t.id?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: lib/customers.js (hunk 1 above)
Comment (RafaelTaranto) [reply lib/customers.js]: at the end this wasn't really needed

Comment (RafaelTaranto) [reply lib/customers.js:457]: The issue is when the subqueries return no results, the count returns as 1 instead of 0

**Pull request merged**